package.json

```diff
@@ -68,5 +68,5 @@
     "sass-embedded-win32-ia32": "1.58.3",
     "sass-embedded-win32-x64": "1.58.3"
   },
-  "packageManager": "pnpm@7.32.0"
-}
\ No newline at end of file
+  "packageManager": "pnpm@v8.10.5"
```

Comment:
you need change engine.pnpm field too

Reply:
We don't have a engine field 🤔

Reply:
it seems we need to rebase 0.4.0 to main first

Reply:
now you have engine field